rsync_tmbackup.sh

```diff
@@ -478,20 +479,16 @@ if [ -n "$(fn_find "$INPROGRESS_FILE")" ]; then
 		fi
 	fi
 
-	if [ -n "$PREVIOUS_DEST" ]; then
+
+	PREVIOUS_DEST_RUNNING="$(fn_find_backups _ | head -n 1)"
+	if [ -n "$PREVIOUS_DEST_RUNNING" ]; then
 		# - Last backup is moved to current backup folder so that it can be resumed.
-		# - 2nd to last backup becomes last backup.
 		fn_log_info "$SSH_DEST_FOLDER_PREFIX$INPROGRESS_FILE already exists - the previous backup failed or was interrupted. Backup will resume from there."
-		fn_run_cmd "mv -- $PREVIOUS_DEST $DEST"
-		if [ "$(fn_find_backups | wc -l)" -gt 1 ]; then
-			PREVIOUS_DEST="$(fn_find_backups | sed -n '2p')"
-		else
-			PREVIOUS_DEST=""
-		fi
-		# update PID to current process to avoid multiple concurrent resumes
-		fn_run_cmd "echo $MYPID > $INPROGRESS_FILE"
+		fn_run_cmd "mv -- $PREVIOUS_DEST_RUNNING $DEST"
```

Comment:
This `mv` moves the incomplete backup to the new name, to resume. It is the same [as before](https://github.com/laurent22/rsync-time-backup/pull/189/files/a5259ce9e4875cb702222d9a10b402abd088d6e3#diff-703e143870d3454e964014cb5854a43eL485) except of the temporary name.